cn/option/series/pie.md

```diff
@@ -169,6 +169,8 @@
 
 {{ target: partial-pie-label }}
 #${prefix} show(boolean) = false
+#${prefix} rotate(string|number) = 0
```

Comment:
The default value of `rotate` should be `false` (see [PieSeries.js](https://github.com/apache/incubator-echarts/blob/502aee669c2ac8c9c2e5db0a01dfcd5c9fbb686d/src/chart/pie/PieSeries.js#L147)), and it's type should be `boolean|number`.